Source/WebCore/platform/graphics/cg/ImageDecoderCG.cpp

```diff
@@ -43,9 +43,17 @@
 #include <ImageIO/ImageIO.h>
 #include <pal/spi/cg/CoreGraphicsSPI.h>
 #include <wtf/FlipBytes.h>
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+#include <wtf/SoftLinking.h>
+#endif
 
 #include "MediaAccessibilitySoftLink.h"
 
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+SOFT_LINK_PRIVATE_FRAMEWORK_FOR_SOURCE(WebCore, PhotosFormats);
+SOFT_LINK_FUNCTION_MAY_FAIL_FOR_SOURCE(WebCore, PhotosFormats, PFMetadataImageSourceIsSpatialMedia, bool, (CGImageSourceRef source), (source));
+#endif
```

Comment:
Can we create a separate `*SoftLink.h/mm` for this?

`DataDetectorsCoreSoftLink` is an example.